lib/db/schema/patch-018-019.sql

```diff
@@ -0,0 +1,199 @@
+
+CREATE TABLE devices (
+  uid BINARY(16) NOT NULL,
+  id INT UNSIGNED NOT NULL,
```

Comment:
I had it as `SMALLINT`but changed it because we *may* want a device id for web sessions at some point and I'd rather never reuse an id (attach detach attach detach) *just in case*. 65k is a lot but not well beyond my imagination for strange user behavior.